site/docs/pages/identity/introduction.mdx

```diff
@@ -16,6 +16,7 @@ OnchainKit provides TypeScript utilities and React components to help you build
 
 The available components are:
 
+- [`<Address />`](/identity/address): Display an account address.
```

Comment:
I am thinking more and more to kill Intro pages and just conver them into Guides.

Nothing to do now. But more something I am thinking more and more.

cc @kyhyco @fakepixels @abcrane123